include/moveit_visual_tools/moveit_visual_tools.h

```diff
@@ -255,6 +255,7 @@ class MoveItVisualTools : public rviz_visual_tools::RvizVisualTools
 
   /**
    * \brief Display an animated vector of grasps including its approach movement in Rviz
+   *        Note this function calls trigger() automatically in order to achieve animations
```

Comment:
this might be confusing as in fact it never calls `trigger` but rather ignores `batch_publish_mode_`

Reply:
```suggestion
   *        Note this function calls publish() automatically in order to achieve animations
```